integrationTests/vm/testInitializer.go

```diff
@@ -464,6 +465,31 @@ func GetIntValueFromSC(gasSchedule map[string]map[string]uint64, accnts state.Ac
 	return big.NewInt(0).SetBytes(vmOutput.ReturnData[0])
 }
 
+func GetIntValueFromSCWithError(gasSchedule map[string]map[string]uint64, accnts state.AccountsAdapter, scAddressBytes []byte, funcName string, args ...[]byte) (*big.Int, error) {
```

Comment:
why is this needed ?

Reply:
It was used instead of `GetIntValueFromSC()` for debugging. `GetIntValueFromSCWithError()` returns extra information if getting the value from the SC fails. Removed now.